subprojects/build-cache-packaging/src/main/java/org/gradle/caching/internal/packaging/impl/TarBuildCacheEntryPacker.java

```diff
@@ -73,6 +74,9 @@
  * Packages build cache entries to a POSIX TAR file.
  */
 public class TarBuildCacheEntryPacker implements BuildCacheEntryPacker {
+
+    private static final String UTF_8 = StandardCharsets.UTF_8.name();
```

Comment:
Naming the field by its value doesn't reflect its intent. What should happen when we decide to use UTF-16? Should we rename the symbol as well, even though the intent is the same?
If you think it makes sense, please consider renaming it.

Reply:
Good point, I changed it.